compose_files/pulp/.gitignore

```diff
@@ -1 +0,0 @@
-pulp-oci-images
```

Comment:
I noticed this directory became an untracked file in git after running the make commands. Why don't we want to ignore it? Is is supposed to be committed?